library/modules/Gui.cpp

```diff
@@ -1601,27 +1384,27 @@ bool Gui::addCombatReport(df::unit *unit, df::unit_report_type slot, int report_
     unit->reports.last_year_tick[slot] = report->time;
 
     switch (slot) {
-        case unit_report_type::Combat:
-            world->status.flags.bits.combat = true;
-            break;
-        case unit_report_type::Hunting:
-            world->status.flags.bits.hunting = true;
-            break;
-        case unit_report_type::Sparring:
-            world->status.flags.bits.sparring = true;
-            break;
+    case unit_report_type::Combat:
+        world->status.flags.bits.combat = true;
+        break;
+    case unit_report_type::Hunting:
+        world->status.flags.bits.hunting = true;
+        break;
+    case unit_report_type::Sparring:
+        world->status.flags.bits.sparring = true;
+        break;
     }
 
     // And all the continuation lines
-    for (size_t i = report_index+1; i < vec.size() && vec[i]->flags.bits.continuation; i++)
+    for (size_t i = report_index + 1; i < vec.size() && vec[i]->flags.bits.continuation; i++)
         rvec.push_back(vec[i]->id);
 
     return true;
 }
 
-bool Gui::addCombatReportAuto(df::unit *unit, df::announcement_flags mode, int report_index)
+bool Gui::addCombatReportAuto(df::unit* unit, df::announcement_flags mode, int report_index)
 {
-    auto &vec = world->status.reports;
+    auto& vec = world->status.reports;
```

Comment:
I'll review this again after the formatting changes are cleaned up. this is breaking my brain : p

Reply:
Sorry ><. I didn't think about editor formatting settings going into this at all and it's biting me in the ass. I'll get it sorted and comment when I do.

Reply:
Formatting stuff fixed up. Still lots of work to do though.